ListViewEx/Backup/ListViewEx.csproj.user

```diff
@@ -0,0 +1,48 @@
+<VisualStudioProject>
```

Comment:
`*.user` and `*.suo` (solution user options) files should be removed from the repo. As well as `UpgradeLog.htm`.